deployment/docker-compose.dev.yml

```
@@ -17,6 +17,33 @@ services:
   # networks:
   #   - xyne
   # restart: always
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)